data/json/items/magazines.json

```diff
@@ -8,12 +8,68 @@
         "description" : "A standard capacity magazine for use with STANAG compatible rifles.",
         "material" : "steel",
         "volume" : 2,
-        "weight" : 110,
-        "to_hit" : -2,
+        "weight" : 360,
         "ammo_type" : "223",
         "capacity" : 30,
-        "reliability" : 10,
-        "reload_time" : 30
+        "reliability" : 8,
+        "reload_time" : 100,
+        "flags" : [ "COMPACT_MAG" ],
+        "alternatives" : [
```

Comment:
Now you have me confused as to whether this property goes on magazine entries, or firearm entries.